src/components/scores/export.cjsx

```diff
@@ -134,6 +136,7 @@ ScoresExport = React.createClass
         isWaiting={ScoresExportStore.isExporting(courseId) or tryToDownload}
         isFailed={ScoresExportStore.isFailed(courseId) or downloadHasError}
         failedProps={failedProps}
+        timeout={3600000}
```

Comment:
I think 10min and just pulling the constant up into the top of the file (or a common place for the job timeout). up to you :smile: